### DISCUSSION TITLE
Animation cropped after updating to 4.4.0

### DISCUSSION THREAD

**Discussion by rockyev** [airbnb/lottie-ios#2315] [Q&A] [answered]
After updating from 3.4.3 I'm having an issue with the animation below.

Expected:

https://github.com/airbnb/lottie-ios/assets/115032508/fe654b23-62c0-42b9-9f1a-b087491d338d

Actual:

https://github.com/airbnb/lottie-ios/assets/115032508/3b7bfa97-fbec-4680-8cbb-4105ab1897cb

If you watch the lower left corner of the leaf being animated, you will see that it is being cropped while moving over to the right. I'll include a freeze frame below for an easier look at it.
<img width="421" alt="Screenshot 2024-02-13 at 1 54 29 PM" src="https://github.com/airbnb/lottie-ios/assets/115032508/74041aba-eb48-4334-83de-4fecb49daa28">

I have confirmed that this issue is only introduced upon updating Lottie, and no changes to the animation have been made.

After updating, I was having an issue with some animations in our snapshot tests that I solved after following the advice given [here](https://github.com/airbnb/lottie-ios/discussions/1769#discussioncomment-3860672). But other than that, this is the only side effect from updating (and this appears to be the only animation that's been affected).

As suggested [here](https://github.com/airbnb/lottie-ios/issues/1946), I've made sure that this animation is being rendered on the main thread but that doesn't make any difference. I've searched for any other posts with this same "cropping" issue but I couldn't find any. Any suggestions/ideas at all would be appreciated! Thank you!

**Comment by calda** [comment]
Please share a copy of your animation or a sample project that demonstrates the issue.

If you can't share the animation publicly, you can email it to me privately at cal.stephens@airbnb.com.

**Comment by calda** [comment, marked as answer]
Oh, I missed that you just updated from 3.x to 4.x.

4.0.0 included this change, which updated `LottieAnimationView` to mask the animation to bounds by default: https://github.com/airbnb/lottie-ios/pull/1814

https://github.com/airbnb/lottie-ios/pull/2008 added a `maskAnimationToBounds` property when you can set to `false` to re-enable the old behavior.
